packages/common/src/hardforks/chainstart.json

```diff
@@ -23,15 +23,7 @@
       "d": "Gas base cost, used e.g. for ChainID opcode (Istanbul)"
     },
     "tierStep": {
-      "v": [
-        0,
-        2,
-        3,
-        5,
-        8,
-        10,
-        20
-      ],
+      "v": [0, 2, 3, 5, 8, 10, 20],
```

Comment:
Sorry for this unnecessary diff, will remove, my linter did it automatically

Reply:
Are you using `npm run lint:fix` on the package?

Reply:
should be fine, as long as it's following our monorepo's prettier rules and not your own local config